src/common/daft-config/src/lib.rs

```diff
@@ -26,6 +26,7 @@ pub struct DaftExecutionConfig {
     pub merge_scan_tasks_max_size_bytes: usize,
     pub broadcast_join_size_bytes_threshold: usize,
     pub sample_size_for_sort: usize,
+    pub split_row_groups_max_files: usize,
     pub num_preview_rows: usize,
```

Comment:
We should also have the filesize threshold to break up a parquet file as a constant here.